pkg/resource/certificate_authority_activation/hooks.go

```diff
@@ -59,8 +56,24 @@ func (rm *resourceManager) customFindCertificateAuthorityActivation(
 		return nil, err
 	}
 
+	if r.ko.Spec.CertificateAuthorityARN == nil && r.ko.Spec.CertificateAuthorityRef != nil {
+		var caResource = schema.GroupVersionResource{Group: "acmpca.services.k8s.aws", Version: "v1alpha1", Resource: "certificateauthorities"}
```

Comment:
Can we reuse the variables declared in https://github.com/aws-controllers-k8s/acmpca-controller/blob/main/pkg/resource/certificate/descriptor.go#L37-L38 - maybe in a follow up PR